docs/tutorials/quick_start_rake_tasks.md

```diff
@@ -0,0 +1,213 @@
+
+    ---
+    HOSTS:
+      ubuntu1404-64-1:
+        pe_dir: 
+        pe_ver: 
+        pe_upgrade_dir: 
+        pe_upgrade_ver: 
+        platform: ubuntu-14.04-amd64
+        hypervisor: vagrant
+        roles:
+        - agent
+        - master
+        - database
+        - dashboard
+        - classifier
+        - default
+      ubuntu1404-64-2:
+        pe_dir: 
+        pe_ver: 
+        pe_upgrade_dir: 
+        pe_upgrade_ver: 
+        platform: ubuntu-14.04-amd64
+        hypervisor: vagrant
+        roles:
+        - agent
+        - frictionless
+    CONFIG:
+      nfs_server: none
+      consoleport: 443
+      box_url: https://vagrantcloud.com/puppetlabs/boxes/ubuntu-14.04-64-nocm
+      box: puppetlabs/ubuntu-14.04-64-nocm
+
+
+For more info on host generation and what these configs represent see - [Creating A Test Environment](creating_a_test_environment.md)
+
+
+### gen_pre_suite
+
+To run:
+
+    rake beaker_quickstart:gen_pre_suite
+    
+This task will generate a default Beaker pre-suite file.
+
+The gen_pre-suite task will create a file 'default_pre_suite.rb' in acceptance/setup.
+
+If the file already exists, it will not be overwritten. This will allow you to edit the pre_suite file yourself if required.
+
+pre_suite file:
+
+    install_puppet
+  
+See the [Test Suites doc](test_suites.md) in this directory for more information on pre-suites.
+
+### gen_smoke_test
+
+To run:
+
+    rake beaker_quickstart:gen_smoke_test
+    
+This task will generate a default Beaker smoke test file.
+
+The gen_smoke_test task will create a file 'default_smoke_test.rb' in acceptance/tests.
+
+If the file already exists, it will not be overwritten. This will allow you to edit the test file yourself if required.
+
+smoke test file:
+
+    test_name 'puppet install smoketest' do
+      step 'puppet install smoketest: verify \'puppet help\' can be successfully called on
+      all hosts' do
+            hosts.each do |host|
+              on host, puppet('help') 
+            end
+      end
+    end
+  
+This smoke test will check that Puppet has been successfully installed on the hosts.
+
+For more information on the Beaker dsl methods available to you in your tests see - [Beaker dsl](../how_to/the_beaker_dsl.md)
+
+
+### Run task
+
+This task will run all the above tasks in sequential order, to generate a hosts file, pre-suite file, smoke test and then use these files
+to perform a Beaker test run. If the files already exist (see below for further info on file names and location) then they will not be
+overwritten.
+ 
+beaker_quickstart:run_test
+
+#### run_test
+
+To run:
+  
+    rake beaker_quickstart:run_test[provisioner]
+    
+This task takes an optional parameter (provisioner), if you leave it empty it will default to 'vagrant'. You can pass in either 'vmpooler' 
+or 'vagrant'.
+    
+If you have the zsh shell then you will need to escape the square brackets:
+    
+    rake beaker_quickstart:run_test\[vmpooler\]

```

Comment:
this is repeated from the section above. Maybe there should just be a "Provisioning Argument" section, so that there's one spot where you explain this? Then down here, you could just say that this task also supports the provisioning argument, rather than needing to explain how that argument works again.

Thanks for putting up with all of this @pinkypie! I know docs can be particularly painful since they go through the most scrutiny, but I think it'll be worth it!